LoopUI/‏‏ar.lproj/Localizable.strings

```diff
@@ -1,32 +0,0 @@
-/* Accessbility format value describing glucose: (1: glucose number)(2: glucose time) */
```

Comment:
Why was this file deleted?

Reply:
It's hard to see in github review here, but there are special characters in the filename. It actually breaks time machine.